Logger/BaseLogger.cs

```diff
@@ -2,6 +2,7 @@
 {
     public abstract class BaseLogger
     {
+        public string className { get; set; }
```

Comment:
Properties should be PascalCase
```suggestion
        public string ClassName { get; set; }
```